From 21d6d6d859f99c5553768f3c6775e4d44b43c124 Mon Sep 17 00:00:00 2001
From: padmashree mandri <padmashree.mandri@intel.com>
Date: Tue, 6 Aug 2024 11:08:39 +0000
Subject: [PATCH] Disbale stub HAL for audio

Reverting back enabling of stub HAL in AOSP
as it is not being supported yet and causing
boot issue.

Tracked-On: OAM-123104
Signed-off-by: padmashree mandri <padmashree.mandri@intel.com>
---
 audio/aidl/default/android.hardware.audio.service-aidl.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/audio/aidl/default/android.hardware.audio.service-aidl.xml b/audio/aidl/default/android.hardware.audio.service-aidl.xml
index c223dc6c00..97e84da0be 100644
--- a/audio/aidl/default/android.hardware.audio.service-aidl.xml
+++ b/audio/aidl/default/android.hardware.audio.service-aidl.xml
@@ -19,11 +19,11 @@
     <version>2</version>
     <fqname>IConfig/default</fqname>
   </hal>
-  <hal format="aidl">
+  <!--hal format="aidl">
     <name>android.hardware.audio.core</name>
     <version>1</version>
     <fqname>IModule/stub</fqname>
-  </hal>
+  </hal-->
   <hal format="aidl">
     <name>android.hardware.audio.core</name>
     <version>1</version>
-- 
2.34.1

